src/components/MenuBar/MenuBar.tsx

```diff
@@ -71,8 +72,34 @@ const MenuBar = ({ useLearningPathTopic = _useLearningPathTopic }: MenuBarProps)
     setAnchorElUser(null)
   }
 
-  const handleOpenTopicsMenu = (event: React.MouseEvent<HTMLElement>) => {
+  const handleOpenTopicsMenu = async (event: React.MouseEvent<HTMLElement>) => {
```

Comment:
weiter unten wieder skeleton als in memo wrappen, weil sich der render nicht ändert